tests/js/spec/views/projectDetail/projectFilters.spec.jsx

```diff
@@ -13,46 +13,34 @@ describe('ProjectDetail > ProjectFilters', () => {
 
   it('recommends semver search tag', async () => {
     const organization = TestStubs.Organization();
-    tagValueLoader.mockImplementationOnce(() =>
-      Promise.resolve([
-        {
-          count: null,
-          firstSeen: null,
-          key: 'release.version',
-          lastSeen: null,
-          name: 'sentry@0.5.3',
-          value: 'sentry@0.5.3',
-        },
-      ])
-    );
-    const wrapper = mountWithTheme(
+    tagValueLoader.mockResolvedValue([
+      {
+        count: null,
+        firstSeen: null,
+        key: 'release.version',
+        lastSeen: null,
+        name: 'sentry@0.5.3',
+        value: 'sentry@0.5.3',
+      },
+    ]);
+    render(
       <OrganizationContext.Provider value={organization}>
         <ProjectFilters query="" onSearch={onSearch} tagValueLoader={tagValueLoader} />
       </OrganizationContext.Provider>,
-      TestStubs.routerContext()
+      {context: TestStubs.routerContext(), organization}
```

Comment:
we discussed using the organization on this object or being explicit and wrapping the component with `<OrganizationContext.Provide` and decided that we would like to go with the second option. Maybe we should remove it from the object some time soon :)

Reply:
Sounds good!